packages/terra-alert/translations/ar.json

```diff
@@ -1,9 +0,0 @@
-{
-  "Terra.alert.dismiss": "تجاهل",
```

Comment:
Talked offline to no request new translations for these types. Should I keep the existing translations even with the missing keys?